classes/spec_extensions.rb

```diff
@@ -85,6 +85,8 @@ def or_license_name_and_url
           return { license: "Apache 2", url: "https://www.apache.org/licenses/LICENSE-2.0.html" }
         elsif license.scan(/apache2/).count > 0
           return { license: "Apache 2", url: "https://www.apache.org/licenses/LICENSE-2.0.html" }
+        elsif license.scan(/apache 2/).count > 0
```

Comment:
`/apache(?: license, version 2(\.0)?|\s*2(\.0)?)/i` is the best I can come up with that matches all of the cases in one regex:

- Apache 2
- Apache 2.0
- Apache2
- Apache2.0

In the following cases, we'd always want a space after "Apache", hence the vertical bar in the regex:
- Apache License, Version 2
- Apache License, Version 2.0